src/platform-includes/session-replay/install/javascript.capacitor.mdx

```diff
@@ -1,7 +1,17 @@
-```bash {tabTitle: Yarn}
-yarn add @sentry/capacitor @sentry/replay
+Install the Sentry Capacitor SDK alongside the corresponding Sentry SDK for the framework you're using, such as Angular in this example:
+
+```bash {tabTitle:Angular}
+# npm
+npm install --save @sentry/capacitor @sentry/angular-ivy @sentry/replay
+
+# yarn
+yarn add @sentry/capacitor @sentry/angular-ivy @sentry/replay
 ```
 
-```html {tabTitle: npm}
+```bash {tabTitle:Other Frameworks}
+# npm
 npm install --save @sentry/capacitor @sentry/replay
+
+# yarn
+yarn add @sentry/capacitor @sentry/replay
 ```
```

Comment:
similar to above, should this be `@sentry/capacitor @sentry/javascript`?

Reply:
capacitor isn't part of the monorepo so I'm not sure if the re-export done works, but possibly so. I'll check and update on a follow up PR

Reply:
cc @lucas-zimerman ^